jobs/build/update-3.x-base-images/Jenkinsfile

```diff
@@ -0,0 +1,51 @@
+node {
+    checkout scm
+    def commonlib = load("pipeline-scripts/commonlib.groovy")
+    def buildlib = load("pipeline-scripts/buildlib.groovy")
+    // Expose properties for a parameterized build
+    properties(
+        [
+            buildDiscarder(
+                logRotator(
+                    artifactDaysToKeepStr: '',
+                    artifactNumToKeepStr: '',
+                    daysToKeepStr: '',
+                    numToKeepStr: '100')
+                ),
+                [
+                $class: 'ParametersDefinitionProperty',
+                parameterDefinitions: [
+                    [
+                        name: 'MAIL_LIST_FAILURE',
+                        description: 'Failure Mailing List',
+                        $class: 'hudson.model.StringParameterDefinition',
+                        defaultValue: [
+                            'aos-team-art@redhat.com',
+                        ].join(',')
+                    ],
```

Comment:
could the job also have a parameter to specify a list of the tags to build? the default would be to do all of them (we'd probably do that on a weekly schedule) but we'd like to be able to just rebuild one or two on demand as well.